src/libtsduck/base/network/tsSRTSocket.cpp

```diff
@@ -346,7 +346,7 @@ TS_MSC_NOWARNING(4668)  // 'xxx' is not defined as a preprocessor macro, replaci
 #include <srt/srt.h>
 
 // On Windows, access_control.h as missing in the binary installer before 1.5.3.
-#if defined(TS_WINDOWS) && SRT_VERSION_VALUE < SRT_MAKE_VERSION_VALUE(1,5,3)
+#if (defined(TS_WINDOWS) && SRT_VERSION_VALUE < SRT_MAKE_VERSION_VALUE(1,5,3)) || SRT_VERSION_VALUE < SRT_MAKE_VERSION_VALUE(1,4,2)
```

Comment:
why?

Reply:
also versioning is weird here

Reply:
access_control.h wasn't added until 1.4.2 https://github.com/Haivision/srt/commit/47fe4eb5aadf3ddef4a31c2b6d21459dfb83af91